src/effects/effectparameterslot.cpp

```diff
@@ -2,35 +2,40 @@
 
 #include "control/controleffectknob.h"
 #include "effects/effectparameterslot.h"
+#include "effects/effectxmlelements.h"
 #include "control/controlobject.h"
 #include "control/controlpushbutton.h"
 #include "controllers/softtakeover.h"
+#include "util/xml.h"
 
 EffectParameterSlot::EffectParameterSlot(const QString& group, const unsigned int iParameterSlotNumber)
         : EffectParameterSlotBase(group, iParameterSlotNumber) {
     QString itemPrefix = formatItemPrefix(iParameterSlotNumber);
     m_pControlLoaded = new ControlObject(
             ConfigKey(m_group, itemPrefix + QString("_loaded")));
+    m_pControlType = new ControlObject(
```

Comment:
They are deleted in the parent EffectParameterSlotBase destructor. Deleting them here causes a segfault. I'll add a comment.